### PR TITLE
Storm day fix

### PR DESCRIPTION
**Pull request by jomey**
Since both of our domains are successfully running with this code, I am suggesting to move this into the main UofU branch.
This is dependent on UofU-Cryosphere/awsm/pull/1, which takes care of the wrong file paths to `storm_days.nc`

Added some tests to the updated logic.

### CODE REVIEW

**Comment by jomey** [comment]
Here an example of new fixed and expected behavior
Storm_days

File: smrf/envphys/storms.py [hunk 1]
```diff
@@ -2,69 +2,67 @@
...
         time_step: Step in days of the model run
-        mass: Threshold for the mass to start a new storm
-        time: Threshold for the time to start a new storm
-        stormDays: If specified, this is the output from a previous run of
-            storms else it will be set to the date_time value
-        stormPrecip: Keeps track of the total storm precip
+        (Optional)
+        mass_threshold: Minimum amount of precipitation required to be a storm
+                        (snow mass). Default: 0.5
+        percent_snow_threshold: Minimum percent snow within a pixel in
```
Comment (dmragar) [inline smrf/envphys/storms.py:37]: I think this is the percent of precip that is _snow_, to account for mixed phase precip. This reads like a spatial metric to me.

**Review by dmragar — APPROVED** [review]

File: smrf/envphys/storms.py (hunk 1 above)
Comment (jomey) [reply smrf/envphys/storms.py:37]: This is the `ps_thresh` parameter from the initial method signature and the threshold marks the value that has to be exceeded for a pixel to be considered as snowfall. I think we agree on that. 

Can you expand on your comment a bit or suggest a different description.

Comment (dmragar) [reply smrf/envphys/storms.py:37]: I suggest: _percent_snow_threshold: Minimum percent of precip that is snow within a pixel in fractional amount. Default: 0.5 (50%)_

But this is subjective—perhaps just related to how I read and think 😁. Your choice!

File: smrf/envphys/storms.py [hunk 2]
```diff
@@ -2,69 +2,68 @@
...
-        time: Threshold for the time to start a new storm
-        stormDays: If specified, this is the output from a previous run of
-            storms else it will be set to the date_time value
-        stormPrecip: Keeps track of the total storm precip
+        (Optional)
+        mass_threshold: Minimum amount of precipitation required to be a storm
+                        (snow mass). Default: 0.5
+        percent_snow_threshold: Minimum fraction for values in 
+                                `percent_snow_precipitation` to be considered 
+                                a snow event. Default: 0.5 (50%)
```
Comment (jomey) [inline smrf/envphys/storms.py:39]: @dmragar - How does this read?

**Pull request merged**